airflow/providers/google/cloud/operators/vertex_ai/custom_job.py

```diff
@@ -1355,6 +1357,26 @@ def __init__(
         self.gcp_conn_id = gcp_conn_id
         self.impersonation_chain = impersonation_chain
 
+    @property
```

Comment:
from where are we going to refer these methods?

Reply:
They serve as aliases for the renamed public properties 'self.training_pipeline_id` and `self.custom_job_id` to avoid breaking